pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -199,6 +199,17 @@ type SchedulerVolumeBinder interface {
 	BindPodVolumes(ctx context.Context, assumedPod *v1.Pod, podVolumes *PodVolumes) error
 }
 
+type PodVolumeClaims struct {
+	// boundClaims are the pod's bound PVCs.
+	boundClaims []*v1.PersistentVolumeClaim
+	// unboundClaimsDelayBinding are PVs that belong to storage classes of the pod's unbound PVCs with delayed binding.
+	unboundClaimsDelayBinding []*v1.PersistentVolumeClaim
+	// unboundClaimsImmediate are the pod's unbound with delayed binding (including provisioning) PVCs.
+	unboundClaimsImmediate []*v1.PersistentVolumeClaim
+	// unboundVolumesDelayBinding are the pod's unbound with immediate binding PVCs (i.e., supposed to be bound already).
+	unboundVolumesDelayBinding map[string][]*v1.PersistentVolume
```

Comment:
```suggestion
	// boundClaims are the pod's bound PVCs.
	boundClaims []*v1.PersistentVolumeClaim
	// unboundClaimsDelayBinding are the pod's unbound with delayed binding (including provisioning) PVCs.
	unboundClaimsDelayBinding []*v1.PersistentVolumeClaim
	// unboundClaimsImmediate are the pod's unbound with immediate binding PVCs (i.e., supposed to be bound already but they are not).
	unboundClaimsImmediate []*v1.PersistentVolumeClaim
	// unboundVolumesDelayBinding are PVs that belong to storage classes of the pod's unbound PVCs with delayed binding.
	unboundVolumesDelayBinding map[string][]*v1.PersistentVolume
```

Reply:
I'm really sorry, I should have been careless when copying and pasting🤦. They should be right now, I double checked.